automatic/7zip.install/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,27 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageName = '7zip.install'
+
+$uninstalled = $false
+[array]$key = Get-UninstallRegistryKey -SoftwareName '7-zip 16.04*'
```

Comment:
Originally my reason for passing the version was to allow users to install multiple 7zip versions side-by-side, but noticed now that it doesn't matter since the registry key is overwritten anyhow.
I'll change it back to not pass the version.